src/Parser/Parts/PartList.cs

```diff
@@ -10,6 +10,23 @@ public PartList()
         {
         }
 
+ 
+        
+        /// <summary>
+        /// Number of <see cref="TextPart"/> with type <see cref="HoleType.Numeric"/>
+        /// </summary>
+        public int NumericPartCount { get; set; }
```

Comment:
It is for the error in Serilog I see

https://github.com/messagetemplates/messagetemplates-csharp/blob/1a49da630a1f9c9144afc280754bf2282a9eaeb0/src/MessageTemplates/MessageTemplate.cs#L60-L78